src/query/service/src/pipelines/processors/transforms/hash_join/transform_hash_join_build.rs

```diff
@@ -49,6 +49,7 @@ pub struct TransformHashJoinBuild {
     input_port: Arc<InputPort>,
     input_data: Option<DataBlock>,
     step: HashJoinBuildStep,
+    step_logs: Vec<HashJoinBuildStep>,
```

Comment:
How do we see these steps.

Reply:
in log file， will be printed by executor